sources/main.cpp

```diff
@@ -17,19 +20,19 @@
 #include "Vector/Vector3.hpp"
 #include "Window.hpp"
 
-int main()
+int demo()
```

Comment:
:(

Reply:
What's wrong? Are you sad because you didn't eat a beignet?

Reply:
i'm not obelix Maadame !

Reply:
you will find your response [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ)